java/piranha/src/test/java/com/uber/piranha/ConfigurationTest.java

```diff
@@ -1626,6 +1628,99 @@ public void exceptionWhenArgumentIndexNotSpecifiedAndNotArgumentIndexOptional()
     flagCleaner.init(b.build());
   }
 
+  private BugCheckerRefactoringTestHelper enumUnchangedTest(BugCheckerRefactoringTestHelper bcr) {
```

Comment:
I still don't quite understand this method or its comment. What is this testing _or_ setting up for other tests?

Reply:
Good point, the `Test` in the name isn't helping here... I'll think of something clearer. Or it may also be cleaner to keep some duplication in tests for readability's sake.